tests/addon/test_integration.py

```diff
@@ -2690,6 +2690,39 @@ def import_note(note_info: NoteInfo) -> AnkiHubImportResult:
                 assert len(import_result.updated_nids) == 0
                 assert len(import_result.deleted_nids) == 0
 
+    def test_import_note_with_missing_fields(
```

Comment:
:wrench: It would be good to have a test where the note has a value in a field and then the fields gets cleared after importing a note where the field was missing.
So the same as here, but the note already exists in Anki and has some text in the "Back" field before the import.